### PR TITLE
Hello! I'd like to help localizing / Japanese text

### PR DESCRIPTION
**Pull request by floatingfragmentz**
Hello sir,

I'd prepared updated text which is in Japanese. and like appreciate if you considered check this.

# note
In Japanese text, some sentence was not translated yet. 
(Maybe, it's remained due to upgrading versions)

Thanks,
RANDO:

### CODE REVIEW

File: translations/base-ja.yaml [hunk 1]
```diff
@@ -281,36 +280,34 @@ ingame:
         title: チュートリアル
         hints:
             1_1_extractor: <strong>抽出機</strong> を <strong>円の形</strong> の上において抽出しましょう！
-            1_2_conveyor: "抽出機を <strong>コンベアベルト</strong> でHUBまで繋げましょう！<br><br> Tip:
-                <strong>マウスのドラッグ</strong> でベルトを引けます。"
+            1_2_conveyor: 抽出機を <strong>コンベアベルト</strong> でHUBまで繋げましょう！<br><br> Tip:
```
Comment (LeopoldTal) [inline translations/base-ja.yaml:283]: This is a syntax error. Put the quotes back in.

File: translations/base-ja.yaml [hunk 2]
```diff
@@ -599,32 +595,30 @@ storyRewards:
...
-            mechanics!<br><br> For the beginning I unlocked you the <strong>Quad
-            Painter</strong> - Connect the slots you would like to paint with on
-            the wires layer!<br><br> To switch to the wires layer, press
-            <strong>E</strong>. <br><br> PS: <strong>Enable hints</strong> in
-            the settings to activate the wires tutorial!"
+        title: ワイヤ＆着色機 (四分割)
+         desc: "<strong>ワイヤーレイヤー</ strong>をアンロックしました。これは通常のレイヤーの上にある別のレイヤーであり、
+         多くの新しいメカニズムが導入されています!<br><br> 最初に<strong> 着色機 (四分割)</ strong>を解除しました
+         - 塗装したいスロットをワイヤーレイヤーに接続してください!<br><br> ワイヤーレイヤーに切り替えるには、
+            <strong>E</strong>キーを押してください。 <br><br>
```
Comment (LeopoldTal) [inline translations/base-ja.yaml:602]: Trailing spaces here and next line

File: translations/base-ja.yaml [hunk 3]
```diff
@@ -599,32 +595,30 @@ storyRewards:
...
             どんなアイテムでも通過させるか、または通過させないかを選ぶこともできます。
     reward_display:
         title: ディスプレイ
-        desc: "You have unlocked the <strong>Display</strong> - Connect a signal on the
-            wires layer to visualize it!<br><br> PS: Did you notice the belt
-            reader and storage output their last read item? Try showing it on a
-            display!"
+        desc: "<strong>ディスプレイ</strong>をアンロックしました - ワイヤーレイヤーの信号と接続して
+            視覚化をしてみましょう!<br><br> PS: ベルトリーダーと保管庫が最後に出力したアイテムを
+            出力していることをお気づきでしたか?
```
Comment (LeopoldTal) [inline translations/base-ja.yaml:613]: Trailing space

**Review by LeopoldTal — CHANGES_REQUESTED** [review]

**Comment by floatingfragmentz** [comment]
Thank you very much for suggesting!!

**Comment by floatingfragmentz** [comment]
An issue of Japanese lang file has solve, but other lang has issue. so please let me know if I have to do. 
Thanks

**Comment by Nucleareal** [comment]
There is two translation PR (#876, #919) based on tobspr's master
And what should I do?

**Comment by tobspr** [comment]
Seems there are conflicts now, can you have a look?

**Comment by Nucleareal** [comment]
This is conficted but many good translations are there, 
and if any reactions are nothing until 11/05/20, I make new PR that contains these (these good translation) based on now master.

**Comment by tobspr** [comment]
Okay closing this now

**Pull request closed without merging**